### PR TITLE
content/docs: Update the Virtualization section

### PR DESCRIPTION
**Pull request by cristian-vijelie**

### CODE REVIEW

File: content/en/docs/concepts/virtualization.md [hunk 1]
```diff
@@ -1,68 +1,86 @@
...
+  An example of Type 2 hypervisor is VirtualBox.
 
+| ![type 1 hypervisor os](/assets/imgs/vm1.svg) | ![type 2 hypervisor os](/assets/imgs/vm2.svg) |
+| :--:									  | :--:								    |
+| Operating systems over type 1 hypervisor				  | Operating systems over type 2 hypervisor 				    |
 
-The hypervisor emulates devices on behalf of the guest OS, including providing
-access to vCPUs, vRAM, VFS' and vNIC(s) which are segmented from the underlying
-hardware.
+### Containers
```
Comment (razvand) [inline content/en/docs/concepts/virtualization.md:35]: Does it make sense to discuss containers? I know it's useful information, but it's not directly tied to Unikraft.

File: content/en/docs/concepts/virtualization.md [hunk 2]
```diff
@@ -1,68 +1,86 @@
...
+This gives it an advantage over classical Operating Systems.
+Being a special type of operating system, Unikraft can run bare-metal or over a hypervisor.
 
-## Supported Platforms in Unikraft
+| ![type 1 hypervisor uk](/assets/imgs/unikraft1.svg) | ![type 2 hypervisor uk](/assets/imgs/unikraft2.svg) |
+| :--: | :--: |
+| Unikraft over Type 1 hypervisor                                                  | Unikraft over Type 2 hypervisor                           		|
 
-Unikraft supports the following type-1 and type-2 hypervisors out-of-the-box:
+The following table makes a comparison between regular Virtual Machines (think of an Ubuntu VM), Containers and Unikernels, represented by Unikraft:
```
Comment (razvand) [inline content/en/docs/concepts/virtualization.md:55]:
```suggestion
The following table makes a comparison between regular virtual machines (think of an Ubuntu VM), Containers and Unikernels, represented by Unikraft:
```

File: content/en/docs/concepts/virtualization.md [hunk 3]
```diff
@@ -1,68 +1,86 @@
...
 
-There is planned or on-going work to provide these platforms to Unikraft:
+Unikraft is usually run in 2 ways:
 
-* `VMWare <#>`_ (see `Github Issue <#>`_); and,
-* `Hyper-V <#>`_ (see `Github Issue <#>`_).
+* As a virtual machine, using `QEMU-KVM` or `Xen`.
+  It acts as an operating system, having the responsibility to configure the hardware components that it needs (clocks, additional processors, etc).
+  This mode gives Unikraft direct and total control over hardware components, allowing advanced functionalities.
+* As a `linuxu` build, in which it behaves as a Linux user-space application.
```
Comment (razvand) [inline content/en/docs/concepts/virtualization.md:70]: Add reference to `linuxu` section.

**Review by razvand — COMMENTED** [review]
Looks good. I added some comments

**Review by razvand — APPROVED** [review]
Looks good to me.

Reviewed-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Review by nderjung — APPROVED** [review]
Thanks!

Approved-by: Alexander Jung <alex@unikraft.org>

**Pull request merged**